test-app/src/message-board.ts

```diff
@@ -376,15 +371,11 @@ interface TestDeleteEvent
   extends EventBusRuleInput<{ postId: string }, "Delete", "test"> {}
 
 const sendNotification = new Function<Notification, void>(
-  new aws_lambda.Function(stack, "sendNotification", {
-    code: aws_lambda.Code.fromInline(`
-exports.handler = async (event) => {
-    console.log('notification: ', event)
-  };
-`),
-    runtime: aws_lambda.Runtime.NODEJS_14_X,
-    handler: "index.handler",
-  })
+  stack,
+  "sendNotification",
+  async (event) => {
+    console.log("notification: ", event);
+  }
 );
```

Comment:
syntax highlighting is broken by the string literal types in the Comment interface - consider moving them to the bottom to aid legibility in GitHub